sdk/cosmos/azure-cosmos/azure/cosmos/aio/_query_iterable_async.py

```diff
@@ -77,11 +78,6 @@ def __init__(
         )
         super(QueryIterable, self).__init__(self._fetch_next, self._unpack, continuation_token=continuation_token)
 
-    async def __aiter__(self):
-        if 'partition_key' in self._options:
-            self._options['partition_key'] = await self._options['partition_key']
-        return self
```

Comment:
Were there any of the other async iterators where we had logic in `aiter` that needs to be moved?
We should also probably remove any other empty `aiter`s if we added them.

Reply:
the other three `aiter` only do `return self` and are also present as `iter` in their sync versions, so I think we should be good with just removing this